scripts/biomedical/Human-GEM_Human1/README.md

```diff
@@ -0,0 +1,153 @@
+* "hierachyType": Enum type, include category such as "partonomy"
+
+"Gene" is connected with  "MetabolicReaction" by property "geneRole", indicating enzyme participating in the reaction.
+
+"MetabolicReaction" and "Metabolite" is connected by "reactant" and "reactionProduct" properties.
+
+"MetabolicReactionSubsystem" class has the following properties:
+* "humanGEMID"
+* "systemBioOntologyTerm", which is also added to the domain of "Gene"
+
+"MetabolicReaction" and "MetabolicReactionSubsystem" is connected by "memberOfMetabolicReactionSubsystem" property.
+
+### Notes and Caveats
+
+Several gene symbols were not abled to map to existing Gene dcid on Data Commons knowledge graph. Specifically, NTAQ1 couldn't be found on GenomeAssembly hg38. NTAQ1, Z83844.1, AC011462.1, AL845331.1, AC022400.5, AP000812.4, AC093512.2, AL160260.1 couldn't be found on GenomeAssembly hg19. <br>
+
+chemblID are used as dcid for metabolites. For metabolites that lack chemblID, chemical formula is used as dcid. Eight metabolites are metabolite classes, in which case humanGEMIDs are used as dcid <br>
+
+### License
+
+Creative Commons Attribution 4.0 International <br>
+
+More information about License can be found [here](https://github.com/SysBioChalmers/Human-GEM/blob/master/LICENSE.md)
+
+## About the Import 
+
+Python scripts file for processing data files are stored in ./script folder <br>
+Please read the description of each file for the appropriate inputs. <br>
+
+### Scripts
+|File| Description|
+---------|-----------
+|format_genes.py|Inputs: genes.tsv, geneRoles.tsv <br> Outputs: formatted genes.csv, geneRoles.csv|
+|format_metabolites.py|Inputs: metabolites.tsv, reactants.tsv, products.tsv <br> Outputs: formatted metabolites.csv, metabolicCellularComparment, reactants.csv, products.csv|
+|format_reactions.py|Inputs: reactions.tsv <br> Outputs: formatted reactions.csv|
+|format_groups.py|Inputs: groups.tsv, groupMemberships.tsv<br> Outputs: formatted groups.csv, groupMemberships.tsv|
+
+### tMCF files
+
+tMCF files are stored in ./tMCF folder <br>
+These files can be used together with the generated .csv files to ingest human1 data to Data Commons. <br>
+
+|File| Description|
+---------|-----------
+|genes.tmcf| combine with genes.csv for ingestion  |
+|metabolites.tmcf| combine with metabolites.csv for ingestion  |
+|reactions.tmcf| combine with reactions.csv for ingestion  |
+|groups.tmcf| combine with groups.csv for ingestion  |
+|geneRoles.tmcf| combine with geneRoles.csv for ingestion  |
+|reactantRoles.tmcf| combine with reactantRoles.csv for ingestion  |
+|productRoles.tmcf| combine with productRoles.csv for ingestion  |
+|groupMemberships.tmcf| combine with groupMemberships.csv for ingestion  |
+|metabolicCellularCompartment.tmcf| combine with metabolicCellularCompartment.csv for ingestion|
+
+### Example 
+
+To format genes.tsv and geneRoles.tsv, put the tsv files and the python script in the same directory and run:
```

Comment:
Can we have a wrapper shell script or something that just runs everything and produces the CSV? It will be easier to run that 4 different commands and managing the different input/output files.